string label
geometry_msgs/PointStamped point_stamped
